### PR TITLE
Add information about limits of opened perf event files.

### PR DESCRIPTION
**Pull request by Creatone**
Signed-off-by: Paweł Szulik <pawel.szulik@intel.com>

### CODE REVIEW

**Comment by iwankgb** [comment]
@bobbypage and this one too. 
I know I'm sending lots of similar comments but these fixes were in limbo for a while.

**Comment by Creatone** [comment]
@bobbypage nice to have this in docs.

File: docs/runtime_options.md [hunk 1]
```diff
@@ -152,6 +152,9 @@ Core perf events can be exposed on Prometheus endpoint per CPU or aggregated by
 - `--disable_metrics="percpu"` - core perf events are aggregated
 - `--disable_metrics=""` - core perf events are exposed per CPU.
 
+It's possible to get "too many opened files" error when a lof of perf events are exposed per CPU. This happens because of passing system limits.
```
Comment (bobbypage) [inline docs/runtime_options.md:155]: nit: typo lof => lot

File: docs/runtime_options.md [hunk 2]
```diff
@@ -152,6 +152,9 @@ Core perf events can be exposed on Prometheus endpoint per CPU or aggregated by
...
+Try to increase it with `ulimit -n <value>`.
```
Comment (bobbypage) [inline docs/runtime_options.md:156]: Can you note that `ulimit -n <value>` increases max number of file descriptions?

**Review by bobbypage — CHANGES_REQUESTED** [review]
two small nits, thanks for updating doc!

File: docs/runtime_options.md (hunk 2 above)
Comment (Creatone) [reply docs/runtime_options.md]: Is it good now?

**Review by bobbypage — APPROVED** [review]
LGTM, thanks!

**Pull request merged**